register.html

```diff
@@ -56,41 +57,46 @@
     <div class="admin-container">
         <!-- <div class="login"> -->
         <!-- <div class="form form-login"> -->
-        <form action="" class="form form-login">
+        <div class="form form-login">
             <div class="form-footer register-field">
-                <input type="text" name="" id="" placeholder="First Name" required>
-                <input type="text" name="" id="" placeholder="Last Name" required>
+                <input type="text" name="" id="firstName" placeholder="First Name">
+                <input type="text" name="" id="lastName" placeholder="Last Name">
 
             </div>
             <div class="form-footer register-field">
-                <input type="text" name="" id="" placeholder="UserName" required>
-                <input type="email" name="" id="" placeholder="Email" required>
+                <input type="text" name="" id="userName" placeholder="UserName">
+                <input type="email" name="" id="email" placeholder="Email">
 
             </div>
             <div class="form-footer register-field">
-                <input type="password" name="" id="" placeholder="Password" required>
-                <input type="password" name="" id="" placeholder="Confirm Password" required>
+                <input type="password" name="" id="password" placeholder="Password">
+                <input type="password" name="" id="confirmPassword" placeholder="Confirm Password">
             </div>
             <div class="form-footer">
-                <input type="submit" value="Register">
+                <input type="submit" value="Register" onclick="register()">
             </div>
             <div class="form-footer">
                 <p>If you already have an account </p>
                 <p> &nbsp;<b> <a href="login.html">SignIn</a></b> </p>
             </div>
-        </form>
-        <!-- </div> -->
-        <!-- </div> -->
-    </div>
-
-
-    <!-- footer -->
-    <footer>
-        <p> Designed and Built by Abraham Kwizera </p>
-    </footer>
-    <!-- end of footer -->
-    <!-- javascript files -->
-    <script src="./js/index.js"></script>
+            </>
+            <!-- </div> -->
+            <!-- </div> -->
```

Comment:
Same here delete commented code

Reply:
Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. allow me to know if changes are seen. 
Yours.